### ISSUE TITLE
cannot load script correctly in html file

### ISSUE THREAD

**Issue by zhaoyao91** [meteor/meteor#6749]
version: meteor 1.3.1

I've loaded some scripts in main.html file, but it doesn't work as expected.
the same code works fine on a single test.html file: 
```
<!DOCTYPE html>
<html>
<head>
  <meta name="viewport"
        content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no">
  <meta charset="utf-8">
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link href="//cdn.bootcss.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
  <link href="//cdn.bootcss.com/summernote/0.8.1/summernote.css" rel="stylesheet">

  <script src="//cdn.bootcss.com/jquery/2.2.1/jquery.min.js"></script>
  <script src="//cdn.bootcss.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <script src="//cdn.bootcss.com/summernote/0.8.1/summernote.min.js"></script>
  <script src="//cdn.bootcss.com/summernote/0.8.1/lang/summernote-zh-CN.min.js"></script>
</head>
<body>
<div id="summernote">Hello Summernote</div>
<script>
  $(document).ready(function () {
    $('#summernote').summernote();
  });
</script>
</body>
</html>
```
but, when the same code is written in the main.html file in a meteor 1.3 project, things go wrong:
```
<head>
  <meta name="viewport"
        content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no">
  <meta charset="utf-8">
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link href="//cdn.bootcss.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
  <link href="//cdn.bootcss.com/summernote/0.8.1/summernote.css" rel="stylesheet">

  <script src="//cdn.bootcss.com/jquery/2.2.1/jquery.min.js"></script>
  <script src="//cdn.bootcss.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
  <script src="//cdn.bootcss.com/summernote/0.8.1/summernote.min.js"></script>
  <script src="//cdn.bootcss.com/summernote/0.8.1/lang/summernote-zh-CN.min.js"></script>
</head>
<body>
<div id="summernote">Hello Summernote</div>
<script>
  $(document).ready(function () {
    $('#summernote').summernote();
  });
</script>
</body>
```
![image](https://cloud.githubusercontent.com/assets/3808838/14374283/cbc39e2a-fd86-11e5-8d5c-0234fb01a317.png)
It seems that these scripts are loaded and run, but the variable $ is not changed, so the tooltip from bootstrap and summernote from summernote are not available in $.

**Comment by Primigenus** [comment]
Take a look at [using onRendered for 3rd party libraries](http://guide.meteor.com/blaze.html#sts=Use onRendered() for 3rd party libraries).

**Comment by laosb** [comment]
You shouldn't use 3rd jQuery in Meteor, as jQuery is a hard dependency.

That difference is easy to explain:
In 1.3, Meteor scripts are moved to the end of `<body>`, so your js files run first, and then comes the Meteor one, which refills variable `$` with the version of jQuery comes with Meteor bundles.

**Comment by laosb** [comment]
@Primigenus he's using `static-html`, so that shouldn't be a solution.

**Comment by zhaoyao91** [comment]
@laosb thanks, you are right. the cdn scripts get loaded first, then the meteor scripts, so the old version of jquery from meteor will override the cdn version of jquery, which has been updated by cdn bootstrap and cdn summernote. there are several ways to solve this problem: 
1. not to use cdn
2. modify meteor jquery package, to use existing jquery variables
both ways work(I've tried), and I finally chose the first one
thanks everyone.
